index.js

```diff
@@ -109,31 +98,25 @@ controller.hears(['hello'], ['message_received', 'direct_message', 'direct_menti
   bot.startConversation(message, askName)
 })
 
-let getUsers = function (bot) {
-  bot.api.users.list({}, (err, response) => {
-    if (err) return err
-    // console.log(response.members[0])
-    response.members.forEach((user) => {
+function getUsers (callback) {
+  web.users.list(token, (err, response) => {
```

Comment:
Because you initialised the slack web client using the token at the top of the file, I think you don't need to pass it in again here. Try it out just in case as I might be wrong...